### PR TITLE
Fixes issue #109: Covering more currency conversions to bitcoin than before.

### PR DESCRIPTION
**Pull request by Manveer695**
Fixes issue #109 i.e. Adding skill to tell latest exchange rates for currencies to bitcoin

Screenshots for the change: 
primage

dream link:
dream.susi.ai/p/currencytobitcoin

### CODE REVIEW

**Review by saurabhjn76 — APPROVED** [review]

**Comment by Manveer695** [comment]
Please review @aliayub007 @chashmeetsingh @betterclever @madhavrathi. Thank you!

File: models/general/knowledge/en/currency_to_bitcoin.txt [hunk 1]
```diff
@@ -53,3 +53,118 @@ convert bitcoin to jpy| convert btc to jpy| convert btc to japanese yen| convert
...
...le | convert btc to rub| convert btc to russian ruble | convert btc to russian ruble | btc to russian ruble |btc to russian ruble |btc to russian ruble |what is latest exchange rate for russian ruble  to bitcoin|what is latest exchange rate for russian ruble to BTC | can you convert bitcoin to russian ruble | can you convert btc to russian ruble | can you convert btc to rub| can you convert btc to rub| btc to rub|can you tell me what is latest exchange rate for rub to bitcoin|can you tell me what is latest exchange rate for rub to btc |could you tell me what is latest exchange rate for rub to btc |could you convert bitcoin to rub| could you convert btc to rub|  could you convert btc to russian ruble | btc to russian ruble |could you tell me what is latest exchange rate for russian ruble to bitcoin|could you tell me what is latest exchange rate for rub to btc | do you know what is latest exchange rate for rub to btc | can you convert bitcoin to rub| can you convert btc to rub| can you convert btc to russian ruble | btc to russian ruble |btc to rub|do you know the latest exchange rate for russian ruble  to bitcoin|do you know what is the latest exchange rate for russian ruble  to btc
```
Comment (AliAyub007) [inline models/general/knowledge/en/currency_to_bitcoin.txt]: There is some redundancy in questions kindly fix that 
like btc to russian ruble |btc to russian ruble

**Review by hpdang — CHANGES_REQUESTED** [review]
@Manveer695 please add everything you have here into the existing bitcoin skill. https://github.com/fossasia/susi_skill_data/blob/master/models/general/knowledge/en/bitcoin.txt 

All bitcoin related stuff will go there. Please do so and delete this file. Thank you.

**Comment by Manveer695** [comment]
@hpdang Ok, mam. I have done the required steps, please review again. Thanks!

**Pull request merged**